hack/verify-steering-election.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+shopt -s extglob
+
+# exclude bios before 2021 since some of them have more than 300 words
+STEERING_ELECTION_BIOS="$(pwd)/events/elections/!(2017|2018|2019|2020)/!(README|RESULTS|voters).md"
+
+invalid_bios=0
+break=$(printf "=%.0s" $(seq 1 68))
+
+for bio in ${STEERING_ELECTION_BIOS} ; do
+  [[ -f $bio ]] || continue
+  word_count=$(wc -w < "$bio")
```

Comment:
Should we word-count the markdown source or the rendered output? For reference, the rendered output of all 2020 candidates was under 300 words.

Reply:
(if it's simpler to use `wc` and it is applied consistently going forward, that's probably fine ... I think the "source vs rendered word count" probably explains some of the confusion around what happened in the 2020 election)

Reply:
+1 to just using `wc`